mmf/modules/decoders.py

```diff
@@ -81,3 +82,124 @@ def forward(self, weighted_attn):
         state["lm_hidden"] = (h2, c2)
 
         return predictions
+
+
+class PyTorchTransformerDecoder(nn.Module):
```

Comment:
Isn't this more of a `VirtextTransformerDecoder` given that Pytorch already has `nn.TransformerDecoder` (albeit without positional embedding)